UPGRADE.md

```diff
@@ -16,9 +16,11 @@ $ make migrate
 
 ## Unreleased
 
+- If you reused any React components in Django templates through `richie-react`, you should know that the `data-locale` attribute on them is no longer used. Instead, they rely on `<html lang>` to pick up their current locale, which should already be set in all documents anyway.
```

Comment:
but most importantly : if you override the richie/base.html template, make sure you port to your template the modification we made on the `lang` attribute of the <html> element.